eslint.json

```diff
@@ -8,13 +8,14 @@
     "mocha"
   ],
   "parserOptions": {
-    "ecmaVersion": 2017
+    "ecmaVersion": 2018
```

Comment:
What is the first version of Node.js that supports ECMAScript version 2018?

Reply:
Object spread properties (prior to version 2018 throws *unexpected token `...`* error) got support since version `8.6.0` of node.js

[node.green - object rest properties](https://node.green/#ES2018-features-object-rest-spread-properties-object-rest-properties)